### PR TITLE
update COCO format documentation

### PR DESCRIPTION
**Pull request by zmaslova**
### Checklist

- [x] I submit my changes into the `develop` branch
- [x] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [x] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [ ] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: cvat/apps/dataset_manager/formats/README.md [hunk 1]
```diff
@@ -536,8 +526,8 @@ archive.zip/
   - `score` (number) - the annotation `score` field
   - arbitrary attributes - will be stored in the `attributes` annotation section
 
-
-*Note*: there is also a [support for COCO keypoints over Datumaro](https://github.com/openvinotoolkit/cvat/issues/2910#issuecomment-726077582)
+Support for COCO tasks via Datumaro is described [here](https://github.com/openvinotoolkit/datumaro/blob/develop/docs/formats/coco_user_manual.md#export-to-coco)
+For example, export COCO keypoints:
```
Comment (zhiltsov-max) [inline cvat/apps/dataset_manager/formats/README.md:530]: The link included useful information about logic of this export, I think, it should be either described here, or the link should be kept.

**Review by zhiltsov-max — CHANGES_REQUESTED** [review]
@azhavoro, do we need to sync with the docs site?

File: cvat/apps/dataset_manager/formats/README.md (hunk 1 above)
Comment (zmaslova) [reply cvat/apps/dataset_manager/formats/README.md]: the link is returned

**Review by zhiltsov-max — APPROVED** [review]

**Comment by nmanovic** [comment]
@zmaslova , could you please update your PR? We have added a special site for our documentation: https://openvinotoolkit.github.io/cvat/docs/

**Comment by zmaslova** [comment]
> @zmaslova , could you please update your PR? We have added a special site for our documentation: https://openvinotoolkit.github.io/cvat/docs/

Updated

**Comment by nmanovic** [comment]
@zmaslova , there is a problem in the patch. Could you please fix it?

image

**Comment by zmaslova** [comment]
> @zmaslova , there is a problem in the patch. Could you please fix it?
> 
> image

Fixed. I also see issues in CHANGELOG.md, I suppose I shouldn't fix them

**Review by nmanovic — APPROVED** [review]

**Pull request merged**